### PR TITLE
Awacs

### PR DESCRIPTION
**Pull request by agcopenhaver**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by agcopenhaver** [comment]
@conda-forge/staged-recipes is this good to go?

File: recipes/awacs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/cloudtools/{{ name }}/archive/{{ version }}.tar.gz"
+  sha256: "9baaf8db8c4e384ac57e3d674130696b6ed2a54a70f8462a827d43da875d0ea8"
+
+build:
+  number: 0
```
Comment (isuruf) [inline recipes/awacs/meta.yaml]: Can this be `noarch: python`?

Comment (agcopenhaver) [reply recipes/awacs/meta.yaml]: It uses 2to3, so I don't think so

Comment (isuruf) [reply recipes/awacs/meta.yaml]: Can you add a TODO here to make it noarch when python2 is dropped?

Comment (agcopenhaver) [reply recipes/awacs/meta.yaml]: Done

**Pull request merged**